File: Task/ViewController.m
//
//  ViewController.m
//  Task
//
//  Created by Elias Pietilä on 18/04/16.
//  Copyright © 2016 Elias Pietilä. All rights reserved.
//

#import "ViewController.h"
#import <Intercom/Intercom.h>
@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}
-(IBAction)openIntercom:(id)sender
{
    [Intercom presentMessageComposer];
}
@end

File: Task/ViewController.swift
//
//  ViewController.swift
//  Task
//
//  Created by Artem Abramov on 26/03/2017.
//  Copyright © 2017 Elias Pietilä. All rights reserved.
//

import UIKit
import Intercom

class ViewController: UIViewController {    
    
    @IBAction func openIntercom(_ sender: Any) {
        Intercom.presentMessageComposer()
    }
 
}
